crates/bones_input/src/time.rs

```diff
@@ -1,4 +1,7 @@
 //! Time functionality for the Bones framework.
+//!
+//! This is a slimmed down version of the [`bevy_time`](https://github.com/bevyengine/bevy/tree/main/crates/bevy_time).
```

Comment:
Tiniest change to make sure that the link will stay valid if Bevy changes or moves things:

```suggestion
//! This is a slimmed down version of the [`bevy_time`](https://github.com/bevyengine/bevy/tree/aa4170d9a471c6f6a4f3bea4e41ed2c39de98e16/crates/bevy_time).
```